rest/model/account/permissions.go

```diff
@@ -62,10 +62,23 @@ type PermissionsMonitoring struct {
 type PermissionsDHCP struct {
 	ManageDHCP bool `json:"manage_dhcp"`
 	ViewDHCP   bool `json:"view_dhcp"`
+	// The fields below are only relevant in DDI v2.5+
+	TagsAllow *[]AuthTag `json:"tags_allow,omitempty"`
+	TagsDeny  *[]AuthTag `json:"tags_deny,omitempty"`
 }
 
 // PermissionsIPAM wraps a User's "permissions.ipam" attribute for DDI.
 type PermissionsIPAM struct {
 	ManageIPAM bool `json:"manage_ipam"`
 	ViewIPAM   bool `json:"view_ipam"`
+	// The fields below are only relevant in DDI v2.5+
+	TagsAllow *[]AuthTag `json:"tags_allow,omitempty"`
```

Comment:
A bit of a nit, but I don't _think_ we need a pointers to slices as an empty slice can evaluate as equal to `nil` which will satisfy the `omitempty` decorator.

Reply:
That's the problem we are trying to solve with the pointer.
* with `omitempty` and not using a pointer an empty slice will always be omited.  This is not what we want as empty slice is a valid value for this field and must be able to be included in a request or recieved in a response.
* without `omitempty` and not using a pointer. `TagsAllow` will _always_ be included in request, when the user does not explicitly define it the zero value will be used (empty slice).  This is also not what we want as this field is not valid in DDI versions below v2.5.

Reply:
I see, thanks for explaining.